### ISSUE TITLE
CSV.decode! add additional \r\n to double quoted value containing \n

### ISSUE THREAD

**Issue by titib** [beatrichartz/csv#88]
CSV.decode! add additional \r\n 

```
{:ok, string_device1} = StringIO.open("\"one\ntwo\nthree\"")
string_device1 |> IO.stream(:line) |> CSV.decode!() |> Enum.each(&IO.inspect/1)
```

The result is 
```
["one\n\r\ntwo\n\r\nthree"]
```

The expected behaviour is : 
```
["one\ntwo\nthree"]
```

Tested on MacOS 10.14 / Elixir 1.6 / OTP 20

**Comment by beatrichartz** [comment]
Hi, this is current behaviour since this library operates line by line. Will fix this in upcoming versions.

**Comment by beatrichartz** [comment]
This is now fixed in `3.0.0`. Please feel free to reopen if issues persist.
